docs/platforms/android/releases/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Mobile SDK Releases
+description: "Learn about Sentry's release channels for Mobile SDK updates."
+sidebar_order: 7000
+---
+
+The Android SDK uses the following channels to release SDK updates:
+
+## `Pre-release`
+
+Pre-releases (alpha, beta, RC,…) are for larger and potentially more impactful changes, such as new features or major versions.
+
+## `Latest`
+
+We continuously release major/minor/hotfix versions from the `main` branch. These releases go through all our internal quality gates and are very safe to use. They're intended to be the default for most teams.
+
+## `Stable`
+
+Releases are promoted from `Latest` to `Stable` when they've been used in the field for some time and at some scale. The length of time a release has been out, the level of adoption, and other quality and stability metrics are considered. These releases will be indicated on the <Link to="https://github.com/getsentry/sentry-java/releases/">releases page</Link> with the `Stable` label.
+
+## FAQ
+
+### Where can I find the latest stable release for any given SDK?
+
+On the <Link to="https://github.com/getsentry/sentry-java/releases/">releases page</Link> of the SDK's GitHub repo. Stable releases will be indicated with the label `(Stable)` in the name of the release. You can also use the search box in the top right to search for the newest releases.
+
+### Does this mean the release channels are not stable?
+
+No. Stability and performance are the top priorities for every release. The `Stable` release status means that the release has been tested and proven to work in the wild and at scale.
+
+### How often do we mark SDKs as stable?
+
+The stable release cycle depends on a number of factors and can vary, but the target cadence is to provide stable releases roughly every month.
+
+### Should I only use stable releases?
+
+Stable releases have been tested in the field and at significant scale. They are intended for a dependable and consistent experience. Please note that the `Stable` release channel will get our newest features later than other releases.
+
+### How does this relate to the stability of a feature?
+
+The `Stable` release status doesn't ascertain the stability of specific SDK or Product features, such as Session Replay. For example, a new feature that is still experimental might be part of a stable SDK release, and it will be opt-in and clearly marked as experimental.
```

Comment:
```suggestion
### How does this relate to the stability of a feature?

A `Stable` release status doesn't guarantee the stability of an SDK or product feature. A new feature that's still experimental might be part of a stable SDK release. Any experimental features will be clearly marked and offered as an opt-in.
```
I want to make sure I'm understanding this correctly - are we saying that stable releases won't always have all Sentry features? Or are we saying that we don't guarantee that all features will be stable?

I'm assuming it's that we don't guarantee that all features will be stable b/c some will be experimental. 

If we're saying that we don't guarantee access to all features, we should say: A `Stable` release status doesn't guarantee the stability of a specific SDK or access to all product features.

Reply:
> I'm assuming it's that we don't guarantee that all features will be stable b/c some will be experimental.

Yes, that's what we're trying to convey here 👍